core/store/src/db/refcount.rs

```diff
@@ -50,8 +50,8 @@ pub(crate) fn merge_refcounted_records(result: &mut Vec<u8>, val: &[u8]) {
 }
 
 /// Returns
-/// (Some(value), rc) if rc > 0
-/// (None, rc) if rc <= 0
+/// (Some(value), rc) if rc >= 0
+/// (None, rc) if rc < 0
```

Comment:
but if rc == 0 the value doesn't exist, so the orginal comment seems correct?

Reply:
Hmmmm, right, we then hit the first branch, and return `(None, 0)`, that makes sesne!